src/ripple/app/misc/ValidatorList.h

```diff
@@ -28,6 +28,7 @@
 #include <ripple/json/json_value.h>
 #include <ripple/overlay/Message.h>
 #include <ripple/protocol/PublicKey.h>
+#include <ripple/rpc/Context.h>
```

Comment:
This violates levelization. I know RPC is included all over the place in the app, but RPC should be at a higher level, so let's not make it worse. Instead, rewrite `getJson` to take a `std::optional<unsigned int> apiVersion` parameter.